internal/app/machined/internal/phase/rootfs/network_configuration.go

```diff
@@ -34,5 +34,10 @@ func (task *NetworkConfiguration) runtime(r runtime.Runtime) (err error) {
 		return err
 	}
 
+	// Create /etc/hosts
+	if err = etc.Hosts(); err != nil {
```

Comment:
Do we need to create this twice? Just above here we do it in `internal/app/machined/internal/phase/rootfs/hostname.go`.